pages/landing/enterprise-eks/pricing/index.html

```diff
@@ -0,0 +1,125 @@
+permalink: /eks-for-enterprise/pricing/
+
+categories:
+  - title: Feature
+    features:
+      - name: IaC Library
+        description: Every Amazon EKS for Enterprise subscription also includes complete access to the Gruntwork Infrastructure as Code Library.
+        plans: ["Standard", "Early Access"]
+      - name: Reference Architecture
+        description: Optionally, you can run enterprise-grade EKS clusters on the Gruntwork Reference Architecture, an end-to-end tech stack, controlled entirely as code, deployed directly to your AWS accounts.
+        available: now
+        plans: ["Standard", "Early Access"]
+      - name: CIS Benchmark
+        description: Optionally, entirely Amazon EKS clusters that have been hardened according to the CIS Benchmark for Kubernetes and CIS Benchmark for Amazon EKS, all defined as Terraform code.
+        plans: ["Standard", "Early Access"]
+      - name: Service Mesh
+        description: "Deploy and configure a service mesh technology of your choice to enable service-to-service communication with security, flexibility, and observability. Examples: Istio, Consul Connect, AWS App Mesh."
+        plans: ["Standard", "Early Access"]
+      - name: Kubernetes Native CI/CD
+        description: "Deploy your apps using GitOps practices with Kubernetes native tooling. Employ techniques such as canary deployments, feature flags, and staggered deployments, with built-in notifications and alerts. Examples: Flux, Argo CD."
+        plans: ["Standard", "Early Access"]
+      - name: Self-service Dashboard
+        description: "Have your team deploy apps using self-service dashboards that track deployment progress and provide error messages right in the UI. Examples: Argo CD Dashboard, Lens."
+        plans: ["Standard", "Early Access"]
+      - name: Metrics Aggregation
+        description: "Instrument your apps and ship metrics to metrics aggregation services. Query metrics along multiple dimensions and create dashboards to view app behavior. Examples: Prometheus, Datadog, CloudWatch."
+        plans: ["Standard", "Early Access"]
+      - name: Log Aggregation
+        description: "Ship all container logs to a central location. Query, inspect, and stream all logs from a single location. Examples: Datadog, Elastic Stack, CloudWatch."
+        plans: ["Standard", "Early Access"]
+      - name: Pluggable deployment strategies
+        description: Deploy your apps using a variety of industry standard deployment techniques, including canary releases and blue/green deploys. Institute automatic rollbacks in the event of a deployment failure or error condition.
+        plans: ["Standard", "Early Access"]
+      - name: Cloud-native Reference Applications
+        description: "Complete reference applications demonstrating how to deploy and optimize various applications.  Examples: Spring boot, Ruby on Rails, Django."
+        plans: ["Standard", "Early Access"]
+      - name: Gruntwork Patcher
```

Comment:
As per my comment above, to keep experiments separate, we prob shouldn't include this

Reply:
Filed #568 to track that this is critical to address prior to launch. Will work on this after merging.

Reply:
Done: 1249a21